lib/ripper-tags/vim_formatter.rb

```diff
@@ -59,6 +59,14 @@ def display_inheritance(tag)
       end
     end
 
+    def display_line_number(tag)
+      if options.line_number && tag[:line]
+        "\tline:%s" % tag[:line]
```

Comment:
I might go to the wrong direction. Actually, what makes `ctags` output the line number is `--fields` option. http://ctags.sourceforge.net/ctags.html

```
−−fields=[+|−]flags
  | n |   | Line number of tag definition
```

So I'm thinking if we could add the option as `--fields+=n` for `ripper-tags`, it would be compatible with ctags, what do you think?